docs/modding/CLI_REFERENCE.md

```
@@ -0,0 +1,246 @@
+```
+✓ Valid — 18 teams, 360 players, 2 competitions, 1 countries, 0 confederations
+```
+
+```
+✗ 3 error(s):
+  teams/bundesliga.json  be.error.package.unknownCountry (entity=fc-koeln, country=DEU)
+  competitions/cups.json be.error.competitionDef.selectorMissingCountry (competition=dfb-cup)
+  package.json           be.error.package.missingId (kind=world)
+```
+
+See the [Error Codes Reference](#error-codes) below for a full list of error codes.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)